docs/src/main/asciidoc/cache.adoc

```diff
@@ -539,19 +539,22 @@ Another way to retrieve a `Cache` instance consists in injecting the `io.quarkus
 ----
 package org.acme.cache;
 
-import jakarta.enterprise.context.ApplicationScoped;
 import jakarta.inject.Inject;
+import jakarta.inject.Singleton;
 
 import io.quarkus.cache.Cache;
 import io.quarkus.cache.CacheManager;
 
 import java.util.Optional;
 
-@ApplicationScoped
+@Singleton
```

Comment:
I'm curious. Is `@Singleton` the new recommended scope by default with Quarkus 3.0? The doc from 2.x still [recommends](https://quarkus.io/guides/cdi#applicationscoped-and-singleton-look-very-similar-which-one-should-i-choose-for-my-quarkus-application) to use `@ApplicationScoped`.

Reply:
We should never be using `@ApplicationScoped` unless there is a good reason to do so. It introduces a proxy object for no good reason

Reply:
Yes I know about that, but with this PR the Cache doc suggests doing the opposite of [the CDI doc](https://quarkus.io/guides/cdi#applicationscoped-and-singleton-look-very-similar-which-one-should-i-choose-for-my-quarkus-application) 😃

Reply:
Yeah, we should fix that too :)